### PR TITLE
[styles] Add charity & second_hand shops

### PR DESCRIPTION
**Pull request by endim8**
Screenshot 2022-08-08 at 23 02 52
Screenshot 2022-08-08 at 22 52 27

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -9777,6 +9777,9 @@ shop-second_hand
 en:4Second Hand
 ru:4Подержанные товары
 
+shop-charity
+en:4Charity Shop
```
Comment (biodranik) [inline data/categories.txt:9781]: Please translate using tools/unix/translate_categories.sh and then fix by using DeepL.

**Review by biodranik — COMMENTED** [review]
1. What are the sources of icons? Are they listed in our copyrights?
2. Please add types_strings translations too.

File: data/categories.txt (hunk 1 above)
Comment (endim8) [reply data/categories.txt:9781]: Ah! That's how you get so many translations! Working on it :)

**Comment by endim8** [comment]
1. The icons are from already attributed sources 👍
2. fortunately, they seem to already be there

fixed all comments

**Review by vng — APPROVED** [review]

**Comment by endim8** [comment]
fixed merge conflicts

**Pull request merged**